ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -1484,5 +1484,263 @@ class SignalSpec: QuickSpec {
 				expect(error).to(equal(TestError.Default))
 			}
 		}
+		
+		describe("combinePrevious") {
+			
+			var sink: Signal<Int, NoError>.Observer!
+			var initialValue: Int!
+			var latestValues: (Int, Int)?
+			
+			beforeEach {
+				latestValues = nil
+				initialValue = 0
+				
+				let (signal, baseSink) = Signal<Int, NoError>.pipe()
+				sink = baseSink
+				signal |> combinePrevious(initialValue) |> observe(next: { latestValues = $0 })
+			}
+			
+			it("should forward the latest value with previous value") {
+				expect(latestValues).to(beNil())
+				
+				sendNext(sink, 1)
+				expect(latestValues?.0).to(equal(initialValue))
+				expect(latestValues?.1).to(equal(1))
+				
+				sendNext(sink, 2)
+				expect(latestValues?.0).to(equal(1))
+				expect(latestValues?.1).to(equal(2))
+			}
+			
+		}
+
+		describe("combineLatest") {
+			
+			var sinkA: Signal<Int, NoError>.Observer!
+			var sinkB: Signal<Int, NoError>.Observer!
+			var sinkC: Signal<Int, NoError>.Observer!
+			var sinkD: Signal<Int, NoError>.Observer!
+			var sinkE: Signal<Int, NoError>.Observer!
+			var sinkF: Signal<Int, NoError>.Observer!
+			var sinkG: Signal<Int, NoError>.Observer!
+			var sinkH: Signal<Int, NoError>.Observer!
+			var sinkI: Signal<Int, NoError>.Observer!
+			var sinkJ: Signal<Int, NoError>.Observer!
```

Comment:
I wrote it (maximum number of arguments) for veryfication.
It's overkill to convey meaning.
It's very very hard :grin: